### ISSUE TITLE
workspaces count down list

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#2799]
[workspace designs](https://www.figma.com/file/NgBjvs0jRbDHhiyn5nicDm/Workspace?node-id=0%3A1)

MVP:

- [x] #2875
- [x] #2801
- [x] #2802
- [ ] Polish UI
- [ ] #2746
- [x] #2831
- [ ] Handle portal session expiration -- If session expires, should we close the workspace list?
- [x] #2846

MVP, but Chuck blocked:

- [ ] Resolve Widget rendering bug: https://github.com/vitessce/vitessce-python/issues/198
- [x] #2850
- [x] #2832
- [x] #2812
- [x] Get both teams on the same page WRT security: See https://github.com/hubmapconsortium/user_workspaces_server/issues/55
- [ ] Use workspace deletion when available: See https://github.com/hubmapconsortium/user_workspaces_server/issues/76
- [ ] #2633: Waiting on https://github.com/hubmapconsortium/user_workspaces_server/issues/28 and https://github.com/hubmapconsortium/gateway/issues/224

MVP, but save for last:

- [x] #2869
- [ ] Implement UI for workspace creation from dataset
- [ ] Add workspaces to the user profile menu

Nice to have:

- [x] #2883
- [ ] From workspaces list, support empty workspace creation
- [x] Update workspace page (polling?)
- [x] #2808
- [ ] ~~Create multiple notebooks in dataset workspace -- Get Nils sign off on this as alternative to forcing early choice on users.~~ Talked, and Nils is unpersuaded.
- [ ] Figure out what a collection notebook looks like
- [ ] Figure out what a search notebook looks like
- [ ] Figure out what a my-lists notebook looks like
- [ ] Notebooks for all entity types, so it's not conditional 
- [ ] scroll shadows... only if it's a fixed height div?
- [ ] Design and implement status icons
- [ ] Design and implement please wait page
- [x] #2848

Definitely out of scope:

- [ ] #2804
